packages/flutter/lib/src/material/flat_button.dart

```diff
@@ -290,6 +290,7 @@ class FlatButton extends StatelessWidget {
       splashColor: _getSplashColor(theme, buttonTheme),
       elevation: 0.0,
       highlightElevation: 0.0,
+      outerPadding: const EdgeInsets.only(top: 6.0, bottom: 6.0),
```

Comment:
Should this only be added if the button itself is sized too small or always?

Reply:
Good point - the spec isn't very clear but only claims a minimum touch target size of 48x48.  I updated outer padding to constraints so we can express this